src/main/scala/fi/aalto/cs/apluscourses/intellij/utils/ModuleUtils.scala

```diff
@@ -34,4 +36,66 @@ object ModuleUtils {
     nonEmpty
   }
 
+  def naiveValidate(@NotNull command: String): Boolean =
+    command.matches("import\\so1\\.[a-z]*(\\_|\\._)$")
```

Comment:
Should we have some special `//TODO` like syntax for O1 specific code?

Reply:
I'm not sure if I fully got you, what did you mean here?

Reply:
It would be good to somehow track what parts of our codebase have O1 specific code that must get ridden of when the plugin is made available to other courses too. My suggestion was that we marked those line for example with
```java
// O1CODE
command.matches("import\\so1\\.[a-z]*(\\_|\\._)$")
```
so `Ctrl+Shift+F O1CODE` would easily list all of them.

Reply:
a bright idea indeed! :bulb: